File: 2st/Yeonwoo/조건 문자열.py
def solution(ineq, eq, n, m):
    answer = 0
    if ineq == '>' and eq == '=' and n >= m:
        answer = 1
    elif ineq == '<' and eq == '=' and n <= m:
        answer = 1
    elif ineq == '>' and eq == '!' and n > m:
        answer = 1
    if ineq == '<' and eq == '!' and n < m:
        answer = 1
    return answer

File: 4st/Hani/programmer/조건 문자열.java
class Solution {
    public int solution(String ineq, String eq, int n, int m) {
        int answer = 0;
        
        String q = ineq + eq;
        
        if (q.equals("<=")) {
            if (n <= m) answer = 1;
        }
        if (q.equals(">=")) {
            if (n >= m) answer = 1;
        }
        if (q.equals(">!")) {
            if (n > m) answer = 1;
        }
        if (q.equals("<!")) {
            if (n < m) answer = 1;
        }
        
        return answer;
    }
}